practice_app/pages/api/getSets.js

```diff
@@ -0,0 +1,47 @@
+const getSets = (request, response) =>
+  new Promise(async (resolve) => {
+    // Imports the Google Cloud client library
+    const vision = require("@google-cloud/vision");
+
+    // Creates a client
+    const client = new vision.ProductSearchClient();
+
+    // Google cloud project id and region name
+    const projectId = "brilliant-era-276800";
+    const location = "europe-west1";
```

Comment:
These constants appear in multiple places. This is a good fit for environment variables.

Reply:
Or maybe a simple file `config.json` that is imported?